change_quotes.py

```diff
@@ -104,6 +105,23 @@ def run_each(self, edit, sel_region):
         if not region:
             return
 
+        if self.custom:
+            fn_name = self.custom[0]
+            try:
+                fn = getattr(self, fn_name)
+            except AttributeError:
+                return
+
+            try:
+                fn_kwargs = self.custom[1]
+                if not isinstance(fn_kwargs, dict):
```

Comment:
I was seeing guards for the existence of the method and for the existence of the second item in the array, so I assumed there should also be a guard for the dict type of this item. Most of the time, when I mess something in the settings, the default values are taking over an everything works.
But we could raise some exception there.